### PR TITLE
bluetooth mapper updates

### LINKED ISSUE #2443: imcompatible of bluetooth mapper and device API v1alpha2
**What happened**:
After upgrade to KubeEdge v1.4, device api has upgrade to v1alpha2, which is imcompatible with current bluetooth mapper.

**What you expected to happen**:
bluetooth mapper works properly after v1.4

**How to reproduce it (as minimally and precisely as possible)**:
create a new bluetooth device after v1.4, bluetooth mapper will not interpret configmap correctly(cannot find propertyVistors)

**Anything else we need to know?**:

**Environment**:
<details><summary>Kubernets Version</summary>

```console
$ kubelet --version
# paste output here
$ kubectl version
v1.4
```

</details>

<details><summary>CloudCore Version</summary>

```console
$ cloudcore --version
v1.4
```

</details>

<details><summary>EdgeCore Version</summary>

```console
$ edgecore --version
# paste output here
```

</details>

**CloudSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
```

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
# paste output here
```

</details>

<details><summary>Go version</summary>

```console
$ go version
# paste output here
```

</details>

<details><summary>Others</summary>
</details>

**EdgeSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
```

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
# paste output here
```

</details>

<details><summary>Go version</summary>

```console
$ go version
# paste output here
```

</details>

<details><summary>Others</summary>
</details>

### PR DESCRIPTION
**Pull request by luogangyi**
**What type of PR is this?**

**What this PR does / why we need it**:
After upgrade to KubeEdge v1.4, device api has upgrade to v1alpha2, which is imcompatible with current bluetooth mapper.

**Which issue(s) this PR fixes**:

Fixes #2443

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: mappers/bluetooth_mapper/configuration/config_map_types.go [hunk 1]
```diff
@@ -29,54 +29,151 @@ const (
...
 
-// DeviceProfile is structure to store in configMap
+// Twin provides a logical representation of control properties (writable properties in the
+// device model). The properties can have a Desired state and a Reported state. The cloud configures
+// the `Desired`state of a device property and this configuration update is pushed to the edge node.
+// The mapper sends a command to the device to change this property value as per the desired state .
+// It receives the `Reported` state of the property once the previous operation is complete and sends
+// the reported state to the cloud. Offline device interaction in the edge is possible via twin
+// properties for control/command operations.
+type Twin struct {
```
Comment (fisherxu) [inline mappers/bluetooth_mapper/configuration/config_map_types.go:39]: Can we reference or reuse these structures from the api?

Comment (luogangyi) [reply mappers/bluetooth_mapper/configuration/config_map_types.go:39]: Let me check the influence by import api

File: cloud/pkg/apis/devices/v1alpha2/device_instance_types.go [hunk 2]
```diff
@@ -287,20 +287,17 @@ type BluetoothReadConverter struct {
...
 type BluetoothOperations struct {
 	// Required: Specifies the operation to be performed to convert incoming data
-	BluetoothOperationType BluetoothArithmeticOperationType `json:"operationType,omitempty"`
+	BluetoothOperationType string `json:"operationType,omitempty"`
 	// Required: Specifies with what value the operation is to be performed
 	BluetoothOperationValue float64 `json:"operationValue,omitempty"`
 }
 
-// Operations supported by Bluetooth protocol to convert the value being read from the device into an understandable form
-type BluetoothArithmeticOperationType string
```
Comment (fisherxu) [inline cloud/pkg/apis/devices/v1alpha2/device_instance_types.go:296]: Let's keep the type definition here according to the code specification :) And when user set value to the BluetoothOperationType, only the values in the const below are allowed.

**Comment by luogangyi** [comment]
I did some tests.
when we import kubeedge api， the size of bluetooth mapper grows from 9MB to 21MB，mem cost from 10M to 13 M
Seems acceptable to me.

File: cloud/pkg/apis/devices/v1alpha2/device_instance_types.go (hunk 2 above)
Comment (luogangyi) [reply cloud/pkg/apis/devices/v1alpha2/device_instance_types.go:296]: ok

**Comment by fisherxu** [comment]
> I did some tests.
> when we import kubeedge api， the size of bluetooth mapper grows from 9MB to 21MB，mem cost from 10M to 13 M
> Seems acceptable to me.

This is a bit weird.. just change the source code of api.

/lgtm
/approve

**Pull request merged**